mods/cnc/rules/defaults.yaml

```diff
@@ -480,10 +480,13 @@
 		SellSounds: cashturn.aud
 	Capturable:
 	WithMakeAnimation:
+		MakeUpgrades: underConstruction
+	DisableOnUpgrade:
+		UpgradeTypes: underConstruction
```

Comment:
> We have yaml strings which are fully lowercase, even for multiple words.

I know, but as penev pointed out, it can get hard to read with all lower-case, though I guess using an underscore instead might be the better solution here.